azurerm/internal/services/mssql/helper/sql_extended_auditing.go

```diff
@@ -42,6 +42,61 @@ func ExtendedAuditingSchema() *schema.Schema {
 	}
 }
 
+func ExpandAzureRmSqlServerBlobAuditingPolicies(input []interface{}) *sql.ExtendedServerBlobAuditingPolicyProperties {
+	if len(input) == 0 {
```

Comment:
```suggestion
	if len(input) == 0 || input[0] == nil {
```
Just some extra panic protection